x/register/keeper/resource_node.go

```diff
@@ -154,7 +154,8 @@ func (k Keeper) AddResourceNodeStake(ctx sdk.Context, resourceNode types.Resourc
 	}
 
 	resourceNode = resourceNode.AddToken(tokenToAdd.Amount)
-	resourceNode.Suspend = false
+	//resourceNode.Suspend = false
```

Comment:
If adding stake doesn't unsuspend the node, then how do we want resource nodes to unsuspend themselves? Should we add a new ppd command to unsuspend that would make SP node send a new slashing tx but with suspend=false and amount=0 ?